extensions/amp-story/1.0/amp-story-tooltip.js

```diff
@@ -109,7 +122,10 @@ export class AmpStoryTooltip {
      * Clicked element producing the tooltip. Used to avoid building the same
      * element twice.
      * @private {?Element} */
-    this.clickedEl_ = null;
+    this.previousClickedEl_ = null;
```

Comment:
nit: I think this is roughly equivalent to what other places call a "target" element; I think it would make sense to use that language here too (for `this.previousClickedEl_` as well as the local vars `clickedEl` below).